optuna/integration/lightgbm.py

```diff
@@ -34,6 +32,8 @@
     setattr(sys.modules[__name__], "LightGBMTuner", tuner.__dict__["LightGBMTuner"])
     setattr(sys.modules[__name__], "LightGBMTunerCV", tuner.__dict__["LightGBMTunerCV"])
 
+    CallbackEnv = object  # NOQA
```

Comment:
Can we not omit this assignment and use the string notation for `CallbackEnv` as well?

Reply:
Sure.